src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int longSide;
+    private int shortSide;
+    private int height;
+
+    public IsoscelesTrapezoid(int longSide, int shortSide, int height, String color) {
+        super(color);
+        this.longSide = longSide;
+        this.shortSide = shortSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) (longSide + shortSide) / 2 * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: "
```

Comment:
Use `System.out.format` or `String.format` for better readability when you have a complex string concatenation like this one.